src/core/nem/external/index.ts

```diff
@@ -0,0 +1,23 @@
+import { Hasher } from '@utils';
+
+interface INacl {
```

Comment:
1. why do you need this interface? 
2. if needed, you should document it.

Reply:
> why do you need this interface?

`nacl-fast.js` is just pure js, by the added interface, we have the typescript's type-safe feature.

Reply:
I added a description for the interface and function.
I'm not sure how about `modL`